client/signup/steps/woocommerce-install/step-business-info/index.tsx

```diff
@@ -176,9 +176,9 @@ export default function StepBusinessInfo( props: WooCommerceInstallProps ): Reac
 						<SupportCard />
 						<StyledNextButton
 							onClick={ () => {
-								dispatch( submitSignupStep( { stepName: 'business-info' } ) );
 								updateOnboardingProfile( 'completed', true );
 								save();
+								dispatch( submitSignupStep( { stepName: 'business-info' } ) );
```

Comment:
This doesn't have any bearing on the situation but moving it down makes some sense.